internal/engine/eval/vulncheck/review.go

```diff
@@ -201,6 +201,8 @@ func (ra *reviewPrHandler) trackVulnerableDep(
 		comment := patch.IndentedString(location.leadingWhitespace, location.line, dep.Dep)
 		body = reviewBodyWithSuggestion(comment)
 		lineTo = len(strings.Split(comment, "\n")) - 1
+	} else if patch.IsNotFound() {
+		body = vulnPkgNotFound
```

Comment:
Need to test if we only find these cases where we can't find any information on the package do we block or not? (I think we should if there is a fixedIn version, but I'm not 100% sure)